docs/sections/papers/ultrafeedback.md

```diff
@@ -46,43 +47,66 @@ from distilabel.steps import (
 )
 from distilabel.steps.tasks import TextGeneration, UltraFeedback
 
+sample_three_llms = sample_n_steps(3)
```

Comment:
Also, can we include that within the `Pipeline` context manager? Or does it need to be defined outside? If there's any reason for that maybe adding a code comment here would be nice!

Reply:
It doesn't matter where you use the function, as a `RoutingBatchFunction` doesn't have/need a `pipeline` argument.

Reply:
It's similar to when you create a new `Step`

Reply:
Oh fair, but in this case it "doesn't need to be" or "cannot be" inside the context manager?